src/Illuminate/Support/ServiceProvider.php

```diff
@@ -134,6 +157,78 @@ protected function publishes(array $paths, $group = null)
     }
 
     /**
+     * Register a view path to be published to the app's configured view directory.
+     *
+     * @param $path
+     * @param $namespace
+     * @param null $group
+     */
+    protected function publishesViews($path, $namespace, $group = null)
+    {
+        $viewPaths = $this->app['config']->get('view.paths', [resource_path('views/vendor/')]);
+        $this->publishes([
+            $path => rtrim($viewPaths[0], '/')."/vendor/$namespace",
```

Comment:
@inxilpro But are you sure that would work? What if somebody where to set two publish view paths at the same time? :) Not sure that's possible but i think this should be considered..